hack/windows/Install-OVS.ps1

```diff
@@ -1,22 +1,32 @@
 Param(
     [parameter(Mandatory = $false)] [string] $DownloadDir,
     [parameter(Mandatory = $false)] [string] $DownloadURL,
-    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch"
+    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch",
+    [parameter(Mandatory = $false)] [bool] $CheckFile = $true,
```

Comment:
How about renaming CheckFile to CheckFileHash?

Reply:
Done.